### ISSUE TITLE
[BUG] Cannot apply new reactions to old messages

### ISSUE THREAD

**Issue by tigattack** [rHomelab/LabBot-Cogs#155]
**Describe the bug**
It seems that when a message is too old and is purged from Red's cache, the reactrole cog can no longer apply reactions to the message as it cannot find it.

**To Reproduce**
1. Find old message
2. Run `[p]reactrole add <messageID> :emote: <roleID>`
3. See error: `Message "<messageID>" not found.`

**Expected behavior**
I would expect this command to be successful.

**Screenshots**
![image](https://user-images.githubusercontent.com/10629864/117463862-03ba9e00-af48-11eb-8331-de55d372a030.png)

**Additional context**
N/A.

**Comment by tigattack** [comment]
As advised by Issy, a functioning workaround to this is to use the message link rather than message ID.

Suggest we either fix this to eliminate the need for a workaround or add a note to the error message and/or help information.

Thoughts @issy?

**Comment by issy** [comment]
As it is a common use-case to request an old/out of cache message in this cog specifically, I think it is worth fixing this, especially as it's a fairly easy fix

**Comment by issy** [comment]
In doing research for this issue I discovered that you can also pass the message argument as `'{channel_id}-{message_id}'` (eg. `360678601227763712-840563181641662509`) which can be retrieved by shift-clicking "Copy ID" on a message in Discord. This is a lot quicker than the bot iterating through guild channels to find the message. I also discovered that message IDs are not unique across channels, sooo that's fun and basically makes this unfeasible
TL;DR - it turns out this isn't a great idea, and 2 workarounds already exist. The more you know!
